### ISSUE TITLE
An error window is displayed after running IMU Inverse kinematics Tool  , but there is no specific information

### ISSUE THREAD

**Issue by Allwillcome** [opensim-org/opensim-core#3067]
I used my own Python script to convert the Xsens dot output file format into the same format as Xsens, and successfully converted the data into a file format readable by OpenSim, and performed sensor calibration, 
![Screenshot - 2021-08-20 21 18 34](https://user-images.githubusercontent.com/35840764/130239417-4d63f5a3-47f1-4f87-a51b-9ccbf622a4fc.png)

When I performed inverse kinematics calculations, An error message window pops up, but there is no specific prompt message. In addition, there is no specific prompt message in the Opensim dialog box.
![Screenshot - 2021-08-20 21 19 30](https://user-images.githubusercontent.com/35840764/130239454-45406988-c65a-4aa4-98be-726b4a129f52.png)

I hope to be able to add specific error messages to tell me what went wrong. So that I can know where I have a problem.
**Basic information of experiment operation:**
 - Windows
 - OpenSim4.3-2021-08-17-354a4C8 and OpenSim 4.3-2021-07-14-1A8D14B
 - I have used the Xsens sensor before and successfully implemented all functions of OpenSense 
 - I used three sensors, on the pelvis，femur, and tibia
 - the orientation and paste position of the sensors were the same as in the official tutorial document

I put the experimental data in  [OneDrive ](https://bsueducn-my.sharepoint.com/:f:/g/personal/waterfree_bsu_edu_cn/Ei3eCLMqxntLti0ULvdxA-oBDl4zdbMPJDWggNMOe5pIhg?e=cwTHao)So that others can reproduce my mistake
- XsensDot2MTWOpenSim  stores xsens dot original data and converted data
- OpenSenseExample_Dot_OneSideLowerLimbs A folder for storing OpenSense raw data and calibration models
Thanks

**Comment by Allwillcome** [comment]
**Additional Information**
When I set the weight of the tibia to 0, I was able to run the model successfully, but when I wanted to run the model again, OpenSim flashed back

**Comment by aymanhab** [comment]
Thanks for reporting, can you save the setting file in GUI and then run from the command line using the opensense executable to see if you get a more detailed error message? OR send the data files to reproduce. Thank you

**Comment by Allwillcome** [comment]
I tried to use a Python script to run the OpenSense function, and no error was reported. I was able to successfully perform inverse kinematics calculations. However, the error message window still pops up in the GUI. I have put the data files in [Google Drive.
](https://drive.google.com/drive/folders/1YN-cilS8Ht7J1LrgvKO2k33WGQ2891Q-?usp=sharing)
There are two folders:
- XsensDot2Mtw4OpenSense                               It is used to convert Xsens Dot data into data that can be used by OpenSim.
- OpenSenseExample_Dot_OneSideLowerLimbs     It is similar to OpenSenseExample, used to perform inverse kinematics of IMU, I saved my operation steps in GUI in the corresponding XML file

In addition, although I can perform inverse kinematics data in the script, the actual action of the model is not a squat. This may have a problem with my data collection process. I will use Xsens Dot to collect data again tomorrow for testing.

**Comment by gilserrancoli** [comment]
Dear,

I see the same "error" using the IMU Inverse Kinematics Tool with the GUI, not with the Matlab API. I obtain the "error" either running the OpenSense example provided or with new captured data. However, I saw that if one opens the "Weights" window and close it (even without doing nothing, i.e. opening the "Weights" window and click on "Close"), this "Error" does not appear and the tool works fine. 
I observed this in OpenSim 4.2 and 4.3 (Windows 10).

Regards,

Gil
